src/components/create-safe/status/usePendingSafeCreation.ts

```diff
@@ -25,6 +25,27 @@ export const pollSafeInfo = async (chainId: string, safeAddress: string): Promis
   })
 }
 
+export const _getTransactionByHash = (provider: JsonRpcProvider, txHash: string) => {
+  return provider.getTransaction(txHash).then((resp) => {
+    if (resp === null) {
```

Comment:
I start to dislike ethers 😅

Reply:
Yep, the function type is definitely an issue as it can return `null`, `undefined` or `Promise<TransactionResponse>` but its only typed as `Promise<TransactionResponse>` 🤦